### ISSUE TITLE
UL dropdown menu  not work inside a treeview correctlly

### ISSUE THREAD

**Issue by taurenshaman** [olton/Metro-UI-CSS#1660]
**Scenario**
Add context menu to each item of TreeView.

**Describe the bug**
The context menu displays in unexpected way.

**To Reproduce**
https://codepen.io/taurenshaman/pen/dyXqGam

**Expected behavior**
The `z-index ` seems not works?

**Screenshots**
normal:  
![image](https://user-images.githubusercontent.com/1308706/98362574-71087780-2068-11eb-94b9-d64f6943c055.png)

changes `position` of the menu via Debug tool:  
![image](https://user-images.githubusercontent.com/1308706/98362547-5fbf6b00-2068-11eb-9118-fae049a81756.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser [Opera, Edge]
 - Metro Version [4.4.0]

**Comment by olton** [comment]
This is a no problem linked with z-index. Add this styles:
```
.treeview .context {
  position: absolute!important;
  right: 0;
  top: 100%; 
}
```

**Comment by olton** [comment]
I fixed this n 4.4.2, and added dropdown services classes `.drop-down-left`, `.drop-down-right`, `.drop-up-left`, `.drop-up-right`.
With 4.4.2, you can add to your dropdown menu class `.drop-down-right` and it all. https://codepen.io/olton/pen/GRqXZLJ?editors=1000
